fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/JobsServletSpec.groovy

```diff
@@ -155,6 +159,28 @@ class JobsServletSpec extends Specification {
         GroovyTestUtils.compareJson(result, expectedResponse, JsonSortStrategy.SORT_BOTH)
     }
 
+    def "jobs endpoint returns filtered results when a valid filter is present"() {
+        setup:
+        String expectedResponse = """{"jobs":[
+                                        {
+                                            "dateCreated":"2016-01-01",
+                                            "jobTicket":"ticket1",
+                                            "query":"https://localhost:9998/v1/data/QUERY",
+                                            "results":"http://localhost:9998/jobs/ticket1/results",
+                                            "self":"http://localhost:9998/jobs/ticket1",
+                                            "status":"success",
+                                            "syncResults":"http://localhost:9998/jobs/ticket1/results?asyncAfter=never",
+                                            "userId": "momo"
+                                        }
+                                  ]}"""
+
+        when: "We send a request to the /jobs endpoint with a valid filters parameter"
+        String result = makeRequest("/jobs", [filters : ["userId-eq[momo]"]])
+
+        then: "We only get the job payload that satisfies the filter"
+        GroovyTestUtils.compareJson(result, expectedResponse, JsonSortStrategy.SORT_BOTH)
+        }
```

Comment:
Should be unindented one level.

It would be nice if we could also at a minimum have a test verifying that we get back the empty list if none of the tickets satisfy the given filter.